record/record.app.js

```diff
@@ -4,6 +4,7 @@
     angular.module('chaise.record', [
         'ngSanitize',
         'chaise.errors',
+        'chaise.filters',
```

Comment:
This doesn't need to be added. This includes two filter functions that are not being consumed by record app. The directives that use it should define it as a dependency.